tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "target": "es2020",
     "module": "commonjs",
     "moduleResolution": "node",
+    "skipLibCheck": true,
```

Comment:
Even after updating api-extractor it has an incompatibility with the lib that's pulled in by TS 4.9. We aren't missing any validation on our code by setting this flag.